### PR TITLE
set asm branch for anthos sample deployment.

### PR DESCRIPTION
**Pull request by zhihansh**

### CODE REVIEW

File: asm/cluster/nodepool.yaml [hunk 1]
```diff
@@ -24,8 +24,19 @@ spec:
...
     machineType: e2-standard-4
+    preemptible: false
+    oauthScopes:
+      - "https://www.googleapis.com/auth/cloud-platform"
+    metadata:
+      disable-legacy-endpoints: "true"
+    serviceAccount: anthos-svc-account@PROJECT_ID.iam.gserviceaccount.com # {"type":"string","x-kustomize":{"setter":{"name":"service-account"}}}
     workloadMetadataConfig:
-      nodeMetadata: GKE_METADATA_SERVER
+      nodeMetadata: EXPOSE
```
Comment (elfinhe) [inline asm/cluster/nodepool.yaml]: Is this EXPOSE required for anthos sample deployment? At least from ASM side, we've moved away from it.

File: asm/project/enable-services.yaml [hunk 2]
```diff
@@ -1,85 +0,0 @@
-# Copyright 2020 Google LLC
```
Comment (elfinhe) [inline asm/project/enable-services.yaml]: why delete this file?

File: asm/cluster/nodepool.yaml (hunk 1 above)
Comment (zhihansh) [reply asm/cluster/nodepool.yaml]: Do you mean the nodeMetadata was EXPOSE and then changed to GKE_METADATA_SERVER in ASM repo? If that's the case, It should be changed to GKE_METADATA_SERVER.

File: asm/project/enable-services.yaml (hunk 2 above)
Comment (zhihansh) [reply asm/project/enable-services.yaml]: The Deployment manager has already enabled all these APIs before installing ASM.

File: asm/cluster/nodepool.yaml (hunk 1 above)
Comment (elfinhe) [reply asm/cluster/nodepool.yaml]: correct.

File: asm/project/enable-services.yaml (hunk 2 above)
Comment (elfinhe) [reply asm/project/enable-services.yaml]: cool

Comment (zhihansh) [reply asm/project/enable-services.yaml]: done

File: asm/cluster/nodepool.yaml (hunk 1 above)
Comment (zhihansh) [reply asm/cluster/nodepool.yaml]: done

**Pull request merged**